docs/kleenex/usages.rst

```diff
@@ -245,37 +246,49 @@ some exceptions.
                                        --clean-devices "[[device_a, device_b, device_c], [device_d, device_e]]"\
                                        --invoke-clean
 
-``-clean_image``
-    specifies images to be used for clean per device
+``-clean_device_image``
+    specifies images to be used for clean per device. See
+    :ref:`the following <kleenex_cli_image_format>` for
+    more details about the expected format of the images and the
+    precedence when images are provided through a combination of
+    `-clean_device_image`, `-clean_os_image`, `-clean_group_image` and `-clean_platform_image`.
 
     .. code-block:: bash
 
-        bash$ kleenex -testbed-file /path/to/my/testbed.yaml\
-                      -clean-file /path/to/my/clean.yaml
-                      -invoke-clean
-                      -clean-image PE1:/path/to/clean_image.bin
+        bash$ kleenex --testbed-file /path/to/my/testbed.yaml\
+                      --clean-file /path/to/my/clean.yaml
+                      --invoke-clean
+                      --clean-device-image PE1:/path/to/clean_image.bin
```

Comment:
I think, with `kleenex` command, arg should be with `-`(single dash) and `_`(underscore)
```suggestion
        bash$ kleenex -testbed_file /path/to/my/testbed.yaml\
                      -clean_file /path/to/my/clean.yaml
                      -invoke_clean
                      -clean_device_image PE1:/path/to/clean_image.bin
```

Reply:
I updated it to `pyats run job jobfile.py`, similar to the description for `-clean_devices`.